hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/CryptoServiceImplTest.java

```diff
@@ -36,14 +37,14 @@ class CryptoServiceImplTest {
 
     private static final String ACCOUNTS = "ACCOUNTS";
     private static final String ALIASES = "ALIASES";
-    private CryptoServiceImpl subject;
+    private StandardCryptoService subject;
```

Comment:
Why is `CryptoServiceImpl` renamed ? Are we removing `CryptoServiceImpl` ?

Reply:
We had a discussion some time ago that it won't be the best idea to call the implementations of our services `*Impl` since we could have more than 1 implementation. Based on that I decided to use `Standard*` as a name definition for the service implementations. I deleted the old dummy and renamed the `CryptoServiceImpl` to `StandardCryptoService`.

Reply:
I am not sure, if this is a good idea. The *Impl name pattern is used all over the place. Having different name patterns is confusing. I think we should discuss this in one of our meetings and use the same name pattern everywhere.

The chances that there is more than one implementation of a service is tiny IMO.

Reply:
we can discuss that again :) I'm just tried to provide an implementation that matches how I understood the discussion. Fine for me to rename the service implementations. But than please let us do this in a seperate PR once this one is merged ;)